e2e/start-companion-with-load-balancer.mjs

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+import { execa } from 'execa'
+import http from 'node:http'
+import httpProxy from 'http-proxy'
+
+const numInstances = 3
+const lbPort = 3020
+const companionStartPort = 3021
+
+// simple load balancer that will direct requests round robin between companion instances
+function createLoadBalancer (baseUrls) {
+  const proxy = httpProxy.createProxyServer({ ws: true })
+
+  let i = 0
+
+  function getTarget () {
+    return baseUrls[i % baseUrls.length]
+  }
+
+  const server = http.createServer((req, res) => {
+    const target = getTarget()
+    // console.log('req', req.method, target, req.url)
+    proxy.web(req, res, { target }, (err) => {
+      console.error('Load balancer failed to proxy request', err.message)
+      res.statusCode = 500
+      res.end()
+    })
+    i++
+  })
+
+  server.on('upgrade', (req, socket, head) => {
+    const target = getTarget()
+    // console.log('upgrade', target, req.url)
+    proxy.ws(req, socket, head, { target }, (err) => {
+      console.error('Load balancer failed to proxy websocket', err.message)
+      console.error(err)
+      socket.destroy()
+    })
+    i++
+  })
+
+  server.listen(lbPort)
+  console.log('Load balancer listening', lbPort)
+  return server
+}
+
+const startCompanion = ({ name, port }) => execa('nodemon', [
+  '--watch', 'packages/@uppy/companion/src', '--exec', 'node', '-r', 'dotenv/config', './packages/@uppy/companion/src/standalone/start-server.js',
+], {
+  stdio: 'inherit',
```

Comment:
If we are giving relative path in the command, we need to specify a cwd.
```suggestion
  cwd: new URL('../', import.meta.url),
  stdio: 'inherit',
```

Reply:
I merged but I think it broke the tests. Any idea why the tests work without your "fix" but not with it?